AEPServices/Sources/ui/UIApplication+Window.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  File.swift
+//  AEPServices
+//
+//  Created by ravjain on 1/11/21.
+//  Copyright © 2021 Adobe. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public extension UIApplication {
```

Comment:
We've discussed this a bit, and we'd prefer to not expose public extensions on types unless we really need to as to not interfere with customers' potential existing extensions.

Is there any way we can make this internal?